index.html

```diff
@@ -364,7 +364,7 @@ <h2>Documentation Conventions</h2>
       </p>
     </section>
 
-  <section id="data-model">
+  <section>
```

Comment:
The auto-generated id for this is `dapt-data-model-and-corresponding-ttml-syntax` - I much prefer the shorter manual one.